src/content/docs/reference/imports.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Import Directives
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import ParameterLayout from '../../../components/parameter_layout.astro';
+import {formats} from '../../lib/formats.ts';
+import FormatTable from '../../../components/format_table.astro';
```

Comment:
Put in some path definitions (see below) and import from `@lib/formats.tsx`, e.g., or from `@components/parameter_layout.astro`.